Services/Certificate/classes/class.ilObjCertificateSettingsGUI.php

```diff
@@ -38,7 +38,7 @@ class ilObjCertificateSettingsGUI extends ilObjectGUI
     /**
      * @var string
      */
-    private $type;
+    protected $type;
```

Comment:
@tfamula I guess we can remove this property completely. IMO it makes no sense that this is declared redundantly in the certificate class.

Reply:
@mjansenDatabay This was actually my first thought, but I was not completely sure about it :+1:

Reply:
@mjansenDatabay Updated!